feature-governance-impl/src/main/res/layout/view_your_delegation.xml

```diff
@@ -18,6 +18,7 @@
             android:id="@+id/viewYourDelegationTracks"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
+            app:title="@string/delegation_tracks"
```

Comment:
It seems we didn't have "Tracks" title all this time starting with the release of delegates
<img width="248" alt="image" src="https://github.com/novasamatech/nova-wallet-android/assets/107959809/46580164-5422-4dbf-817d-9e8d90e609e6">